src/python/pants/backend/experimental/python/lockfile_test.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from unittest.mock import MagicMock
+
+from pants.backend.experimental.python.lockfile import (
+    PythonLockfileRequest,
+    lockfile_metadata,
+    read_lockfile_metadata,
+    validated_lockfile_content,
+)
+from pants.backend.python.util_rules.interpreter_constraints import InterpreterConstraints
+from pants.util.ordered_set import FrozenOrderedSet
+
+
+def test_metadata_round_trip():
+
```

Comment:
Formatting nit: why the new line for each test?

Reply:
it's a long-term personal habit that `black` doesn't seem to complain about :)

Removing!